packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -143,31 +143,10 @@ class _CupertinoTextFieldSelectionGestureDetectorBuilder extends TextSelectionGe
 /// [controller]. For example, to set the initial value of the text field, use
 /// a [controller] that already contains some text such as:
 ///
-/// {@tool snippet}
+/// {@tool dartpad}
+/// This sample shows how to provide initial text to a CupertinoTextField.
```

Comment:
I'd probably just move the description from the last sentence in the above paragraph into this spot:

```
/// This example shows how to set the initial value of the `CupertinoTextField` using
/// a [controller] that already contains some text.
```